### PR TITLE
Getting CLI working again

### PR DESCRIPTION
**Pull request by begonaguereca**
🟢 🟢 🟢 

We have a couple of changes that happened in order to make this PR work: 

1. We moved our EMU orgs to point to one that Ocotoshift has control over:
https://github.com/enterprises/fabrikam/sso

OnePass has been updated ✅ 

2. Since we moved over to the new EMU org, there are lots of other existing "groups" there, so we had to fix a pagination bug. The issue was addressed in [this PR](https://github.com/github/gh-gei/pull/1277) and the changes were pulled into this branch.

3. Dynamically naming the BBS tests fixes the following error: 
```
FAILURE_REASON: Git source migration failed. Error message: A migration with the same target ssh url has already been queued.
```

4. The Flaky BBS tests we have uncovered is a bug that the Git systems team is addressing here: https://github.com/github/repos/issues/12947#issuecomment-2391937931

My recommendation is that we move forward for now, catch the CI tests at the right time and let the Git systems team address that bug. 

Closes: https://github.ghe.com/github/octoshift/issues/9379
Closes: https://github.ghe.com/github/octoshift/issues/9347
Closes: https://github.ghe.com/github/octoshift/issues/9373
Closes: https://github.ghe.com/github/octoshift/issues/9378

### CODE REVIEW

File: src/OctoshiftCLI.IntegrationTests/BbsToGithub.cs [hunk 1]
```diff
@@ -87,10 +89,10 @@ await retryPolicy.Retry(async () =>
...
 
             await sourceHelper.CreateBbsProject(bbsProjectKey);
-            await sourceHelper.CreateBbsRepo(bbsProjectKey, "repo-1");
-            await sourceHelper.InitializeBbsRepo(bbsProjectKey, "repo-1");
-            await sourceHelper.CreateBbsRepo(bbsProjectKey, "repo-2");
-            await sourceHelper.InitializeBbsRepo(bbsProjectKey, "repo-2");
+            await sourceHelper.CreateBbsRepo(bbsProjectKey, repo1);
+            await sourceHelper.InitializeBbsRepo(bbsProjectKey, repo1);
+            await sourceHelper.CreateBbsRepo(bbsProjectKey, repo2);
+            await sourceHelper.InitializeBbsRepo(bbsProjectKey, repo2);
```
Comment (ArinGhazarian) [inline src/OctoshiftCLI.IntegrationTests/BbsToGithub.cs:95]: Not sure why this change is necessary, the `repo1` and `repo2` variables are the names of target repos in GitHub the hardcoded ones "repo-1" and "repo-2" are the BBS repo names. By default the target repo name in GitHub for BBS migrations is the combination of the `bbsProjectKey` and the BBS repo name so for example for `repo-1` in BBS the target repo name would be `bbsPorjectKey-repo-1`.

**Review by ArinGhazarian — CHANGES_REQUESTED** [review]

File: src/OctoshiftCLI.IntegrationTests/BbsToGithub.cs [hunk 2]
```diff
@@ -4,6 +4,7 @@
 using System.Net.Http;
 using System.Text.RegularExpressions;
 using System.Threading.Tasks;
+using System.Threading;
```
Comment (begonaguereca) [inline src/OctoshiftCLI.IntegrationTests/BbsToGithub.cs:7]: this

File: src/Octoshift/Services/GithubApi.cs [hunk 3]
```diff
@@ -574,11 +574,10 @@ public virtual async Task<int> GetIdpGroupId(string org, string groupName)
     {
         var url = $"{_apiUrl}/orgs/{org.EscapeDataString()}/external-groups";
 
-        // TODO: Need to implement paging
-        var response = await _client.GetAsync(url);
-        var data = JObject.Parse(response);
+        var response = await _client.GetAllAsync(url)
```
Comment (begonaguereca) [inline src/Octoshift/Services/GithubApi.cs:577]: Here

File: src/OctoshiftCLI.IntegrationTests/BbsToGithub.cs (hunk 1 above)
Comment (begonaguereca) [reply src/OctoshiftCLI.IntegrationTests/BbsToGithub.cs:95]: These changes have resolved the following error: 
```
FAILURE_REASON: Git source migration failed. Error message: A migration with the same target ssh url has already been queued.
```

Table showing all the CI runs: https://github.ghe.com/github/octoshift/issues/9347#issuecomment-8651851

**Review by ArinGhazarian — APPROVED** [review]
🎉 🥇
